client/src/components/registration/RegistrationForm.js

```diff
@@ -89,6 +105,26 @@ const RegistrationForm = () => {
     <div className="grid-container">
       <h1>Register</h1>
       <form onSubmit={onSubmit}>
+        <div>
+          <label>First name
+            <input type="text"
+            name="firstName"
+            value={userPayload.firstName}
+            onChange={onInputChange}
+            />
+            <FormError error={errors.firstName} />
+          </label>
+        </div>
+        <div>
+          <label>Last name
+            <input type="text"
+            name="lastName"
+            value={userPayload.lastName}
+            onChange={onInputChange}
+            />
+            <FormError error={errors.lastName} />
+          </label>
+        </div>
```

Comment:
I know we may have merged this in previously, but the divs around each of these input dont actually serve any purpose. Delete them unless you want to target them with styling